azure-pipelines.yml

```diff
@@ -338,3 +351,4 @@ jobs:
     - ${{ parameters.pythonBuildSteps }}
     - ${{ parameters.pythonCoverageSteps }}
     - ${{ parameters.pythonMacWheelSteps }}
+    - ${{ parameters.pythonSDistSteps }}
```

Comment:
we use os x to dist but other than that no reason

Reply:
ah and our linux builds occur within a specialized docker image

Reply:
Why does the docker container affect sdist?  Asking for a friend.

Reply:
Sorry, not a friend- asking because the OSX builds fail 50% of the time for Azure-having-a-moment reasons and I am keenly interested in de-prioritizing them to save myself some sanity.

Reply:
it shouldn't in theory, but I can't just run the bash script in that case